config/routes.rb

```diff
@@ -248,6 +248,7 @@
 
   scope '/services' do
     mount VBADocuments::Engine, at: '/vba_documents'
+    mount VaFacilities::Engine, at: '/openid_auth'
```

Comment:
VaFacilities copy/paste ;)

Reply:
Also I'm not sure we'll want this under /services or if it will wind up under another path like /internal or something like that, as it's geared towards API providers within VA, not API consumers. But fine for now, just a heads up the enclosing scope may change.

Reply:
ya I caught the copy paste problem right after I pushed the initial branch cause it failed when I ran rspec, I can create the /internal scope now if you'd like?